pkg/types.go

```diff
@@ -248,6 +256,8 @@ func (t Type) String() string {
 		return "pip"
 	case Ruby:
 		return "gem"
+	case Rust:
+		return "cargo"
```

Comment:
I think this should return "rust" because the docs say that Rust modules have `type: rust` in `.fossa.yml`. When you run `fossa init` and see:
```
analyze:
  modules:
  - name: Cargo.lock
    type: cargo
    target: .
    path: .
```
it might appear a little surprising that `type` isn't `rust` as the docs suggest it should be. On the other hand, it'll be just as easy to change the docs

Reply:
Thanks for the catch, fetcher type of `cargo` is actually correct, but I will change the docs.

Reply:
buildtools/cargo/testdata/Cargo.lock